spec/controllers/assignments_controller_spec.rb

```diff
@@ -66,6 +66,23 @@
         expect(flash[:error]).to eql('Invalid repository name, use the format owner/name')
       end
     end
+
+    context 'repo_id for starter_code is passed' do
```

Comment:
You need one more context here `invalid repo_id for starter_code is passed`, this could be the case of a repository the user doesn't have access to or an invalid input for `repo_id` like the string `abc` or `0`.